xtask/src/lib.rs

```diff
@@ -155,7 +155,7 @@ pub fn build_documentation(
         .subcommand("doc")
         .arg("-Zbuild-std=core") // Required for Xtensa, for some reason
         .target(target)
-        .features(&[chip.to_string()]);
+        .features(&[chip.to_string(), "ci".to_owned()]);
```

Comment:
This feature should only be added for `esp-hal`, since other packages (like `esp-lp-hal`) do not have this feature.